app/views/users/show.html.erb

```diff
@@ -1,2 +1,20 @@
-<%= @user.first_name %>
-<%= @user.last_name %>
\ No newline at end of file
+<%= render 'shared/side-bar' %>
+<div class="container">
+  <div class="card-detail">
+    <h1> <%= @user.first_name.titleize %> <%= @user.last_name.titleize %></h1>
+    <h2> <%= @user.job_title %></h2>
+    <div class="card-detail-body">
+      <div class="card-detail-info">
+        <ul>
+          <% @user.trips.each do |trip| %>
+            <p>Destination : <%= trip.destination %></p>
+            <p>Purpose : <%= trip.purpose  %></p>
+            <p>Customer : <%= trip.customer %></p>
+            <% end %>
+        </ul>
+      </div>
+    </div>
+  </div>
+    <%= link_to "Go Back", trip_path, class: "btn btn-approve w-30" %>
```

Comment:
`trip_path` wouldn't work without `id`

also are you sure going back to `trip` and not `dashboard`?